packages/babel-preset-react-app/index.js

```diff
@@ -61,14 +53,42 @@ if (env === 'development' || env === 'test') {
     require.resolve('babel-plugin-transform-react-jsx-self')
   ]);
 }
-if (env === 'production') {
-  // Optimization: hoist JSX that never changes out of render()
-  // Disabled because of issues:
-  // * https://github.com/facebookincubator/create-react-app/issues/525
-  // * https://phabricator.babeljs.io/search/query/pCNlnC2xzwzx/
-  // * https://github.com/babel/babel/issues/4516
-  // TODO: Enable again when these issues are resolved.
-  // plugins.push.apply(plugins, [
-  //   require.resolve('babel-plugin-transform-react-constant-elements')
-  // ]);
+
+if (env === 'test') {
+  module.exports = {
+    presets: [
+      [require('babel-preset-env').default, {
+        "targets": {
+        "node": parseInt(process.versions.node),
```

Comment:
1. Let's change this to `parseFloat` as we discussed to fix features introduced in minor Node releases.
2. This line misses an indentation.